### PR TITLE
Handle Enter keys on the iframe side.

### PR DESCRIPTION
**Pull request by jaredhirsch**
@nchapman thoughts?

### CODE REVIEW

File: src/views/index_view.js [hunk 1]
```diff
@@ -31,6 +31,16 @@ export default BaseView.extend({
...
     } else if (data.key === 'ArrowUp' || (data.key === 'Tab' && data.shiftKey)) {
       this._selectNextItem(-1);
+    // enter triggers a navigation
+    } else if (data.key === 'Enter') {
+      let selectedItem = this.query('li.selected');
+      if (selectedItem) {
+        // trick the element into navigating
+        // TODO: use JS events, not DOM events, for signaling
+        var fakeClick = new CustomEvent('mousedown');
+        fakeClick.which = 1;
```
Comment (jaredhirsch) [inline src/views/index_view.js]: The `mousedown` handlers check that it was a left mouse key (`which = 1`), so we have to mock that out here.

It's certainly less than ideal to handle keys by overloading click handlers, but it does reliably propagate the right signals back and forth, so hopefully we can merge as-is and rethink the API soon.

**Comment by jaredhirsch** [comment]
I've landed some backwards-incompatible changes in the addon repo that match this change (https://github.com/mozilla/universal-search-addon/pull/54), so we'll want to deploy both pieces at roughly the same time.

File: src/views/index_view.js (hunk 1 above)
Comment (nchapman) [inline src/views/index_view.js]: Yeah this seems fine for now. What if we move [this](https://github.com/mozilla/universal-search-content/blob/handle-enter-on-frame/src/views/index_view.js#L36-L42) into its own function and create an issue to remind us to replace it?

Comment (jaredhirsch) [inline src/views/index_view.js]: Sounds good, updated the code and filed #37 to track the issue.

**Pull request merged**